routes/drones.js

```diff
@@ -2,35 +2,66 @@ const express = require('express');
 const router = express.Router();
 
 // require the Drone model here
+const Drone 		= require("./../models/Drone.model")
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+router.get('/drones', async (req, res, next) => {
+  const allDrones = await Drone.find({})
+  res.render("drones/list", {
+		data: allDrones
+	})
 });
 
-router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.get('/drones/create', async (req, res, next) => {
+  res.render("drones/create-form")
 });
 
-router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.post('/drones/create', async (req, res, next) => {
+  const image = req.body.image
```

Comment:
req,body es un objeto entonces podemos destructurar estas propiedades con su valor de la sguiente manera 👇 
```suggestion
  const {image, name, propellers, maxSpeed} = req.body
```